class/defaults.yml

```diff
@@ -37,6 +37,27 @@ parameters:
           type: fluentd
           fluentd: {}
 
+    clusterLogForwarding:
+      enable: false
```

Comment:
Personally, I'd call this parameter `enabled`, but I don't really care one way or the other.

Reply:
Is fine by me, changed it.